.github/workflows/ci.yml

```diff
@@ -223,20 +230,24 @@ jobs:
         with:
           name: fluvio-run-x86_64-unknown-linux-musl
           path: ~/.fluvio/extensions
-      - run: chmod +x ~/.fluvio/extensions/fluvio-run
       - name: Download artifact - flv-test
         uses: actions/download-artifact@v2
         with:
           name: flv-test-x86_64-unknown-linux-musl
           path: .
       - name: Print artifacts and mark executable
-        run: ls -la . && chmod +x ./fluvio ./flv-test && ./fluvio version && ./flv-test -h
+        run: ls -la . && chmod +x ./fluvio ./flv-test && ./fluvio -h && ./flv-test -h
```

Comment:
This will now fail since we are cross-compiling more targets. Attempting to invoke `./fluvio` when it is compiled for `armv7-unknown-linux-gnueabihf` will cause a failure.

Reply:
But, for this test we are specifically downloading `fluvio-x86_64-unknown-linux-musl`

Reply:
Ah I see, for some reason I thought this was in the `build_binareis` job, but I was reading the `needs: build_binaries`. In that case, looks good 👍